cmr-exchange/ses-plugin/src/cmr/sizing/formats.clj

```diff
@@ -90,13 +90,6 @@
     (* total-dimensionality
        (max 2 (util/data-type->bytes data-type)))))
 
-(defn- get-avg-gran-size
-  "Gets SizeEstimation value for AverageSizeOfGranulesSampled and parses it to a number."
-  [variable]
-  (-> variable
-      (get-in [:umm :SizeEstimation :AverageSizeOfGranulesSampled])
```

Comment:
Can I get rid of AverageSizeOfGranulesSampled from the UMM-Var schema since it is no longer being used?

Reply:
Yes.